src/Shared/TagTransformerJsonHelper.cs

```diff
@@ -0,0 +1,64 @@
+// <copyright file="TagTransformerJsonHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Text.Json;
+#if NET6_0_OR_GREATER
+using System.Text.Json.Serialization;
+#endif
+
+namespace OpenTelemetry.Internal;
+
+/// <summary>
+/// This class has to be partial so that JSON source generator can provide code for the JsonSerializerContext.
+/// </summary>
+#pragma warning disable SA1601 // Partial elements should be documented
```

Comment:
nit: If you changed this to just a comment and not a "summary" it might make SA1601 go away, and you wouldn't need the suppression.

Reply:
Removed the disable SA1601 (version control issue on my end) - 
/// <summary> does work for SA1601.